src/baseline_automatic_pipeline_system/parse_okr_v2.py

```diff
@@ -0,0 +1,272 @@
+run it from baseline_automatic_pipeline_system directory.
+
+steps:
+1. get list of sentences (tweets).
+2. parse each sentence to get single-sentence semantic representation
+3. combine: create combined lists of EntityMentions and of PropositionMentions (=predicate template is a proposition-mention)
+4. use baseline coref system to cluster EntityMentions to Entities
+5. use baseline coref system to cluster PropositionMentions to Propositions
+6. create okr object based on the pipeline results
+7. evaluate automatic-pipline results
+8. generate output json
+
+"""
+
+import sys, logging
+from docopt import docopt
+sys.path.append("../common")
+sys.path.append("../baseline_system")
+
+from parsers.props_wrapper import PropSWrapper
+import okr
+
+# TODO handle input (tweets raw data)
+# instead of step 1 - meantime suppose its just a simple text file of line-separated sentences, given in tweets_file
+tweets_file = "../../examples/raw_sentences"
+def get_raw_sentences_from_file(input_fn):
+    """ Get list of raw sentence out of file. ignore comments. """
+    raw_sentences = []
+    for line in open(input_fn):
+        sent = line.strip()
+        if not sent or sent.startswith('#'):
+            # Ignore commented lines
+            continue
+        raw_sentences.append(sent)
+
+    # make it a dict {id : sentence}, id starting from 1
+    sentences_dict = {"S"+str(id) : sentence for id, sentence in enumerate(raw_sentences, start=1)}
+    return sentences_dict
+
+# step 2
+def parse_single_sentences(raw_sentences):
+    """ return a list of parsed single-sentence representation. uses props as parser."""
+    pw = PropSWrapper(get_implicits=False,
+                      get_zero_args=False,
+                      get_conj=False)
+    parsed_sentences = {}
+    for sent_id, sent in raw_sentences.items():
+        pw.parse(sent)
+        parsed_sent = pw.get_okr()
+        parsed_sentences[sent_id] = parsed_sent
+    return parsed_sentences
+
+#step 3
+def get_mention_lists(parsed_sentences):
+    """ return combined lists of EntityMentions and of PropositionMentions. """
+
+    all_entity_mentions = {}        # { global-unique-id : entity-mention-info-dict }
+    all_proposition_mentions = {}   # { global-unique-id : proposition-mention-info-dict }
+                                    # global-unique-id is composed from sent_id + "_" + key(=id-symbol at props_wrapper)
+
+    for sent_id, parsed_sent in parsed_sentences.items():
+        # *** entities ***
+        # mentions_keys_terms_indices = [(key, unicode(terms), indices, sent_id) for key, (terms, indices) in parsed_sent["Entities"].items()]
+        # # each mention is a (key, id, terms) triplet
+        # sentence_entity_mentions = [(mention_info[0], id)+ mention_info[1:] for id, mention_info in zip(mentions_ids, mentions_keys_terms_indices)]
+        # # add all entity-mentions of sentence to global list
+        # all_entity_mentions.extend(sentence_entity_mentions)
```

Comment:
Is this commented-out code? 
If so, better remove it, so it reduces clutter. It's always backed up in your codebase, as you've pushed this, in case you want to get back to it.